### ISSUE TITLE
Misleading log message from validator before genesis start

### ISSUE THREAD

**Issue by xrow** [prysmaticlabs/prysm#5452]
I see "Waiting for beacon node to sync to latest chain head" even the chain is in sync with head before genesis. The message is misleading. It would be changed or improved.

```
time="2020-04-16 05:48:18" level=info msg="Beacon chain started" genesisTime=2020-04-18 02:00:00 +0200 CEST prefix=validator
--
  | time="2020-04-16 05:48:24" level=info msg="Waiting for beacon node to sync to latest chain head" prefix=validator
  | time="2020-04-16 05:48:30" level=info msg="Waiting for beacon node to sync to latest chain head" prefix=validator
  | time="2020-04-16 05:48:36" level=info msg="Waiting for beacon node to sync to latest chain head" prefix=validator
  | time="2020-04-16 05:48:42" level=info msg="Waiting for beacon node to sync to latest chain head" prefix=validator
  | time="2020-04-16 05:48:48" level=info msg="Waiting for beacon node to sync to latest chain head" prefix=validator
```

**Comment by prestonvanloon** [comment]
This should be fixed with #5396. @0xKiwi can you confirm?

**Comment by 0xKiwi** [comment]
Interesting, this log shouldn't be appearing before chainstart occurs, but #5396 won't affect this. However, #5465 will change the validator to no longer ping the beacon node every 6 seconds.
